packages/noya-state/src/reducers/canvasReducer.ts

```diff
@@ -188,17 +187,18 @@ export function canvasReducer(
           draft.sketch.pages[pageIndex].layers.push(layer);
         }
 
-        const encodedPoint: Sketch.CurvePoint = {
+        const decodedPoint: DecodedCurvePoint = {
           _class: 'curvePoint',
           cornerRadius: 0,
-          curveFrom: stringifyPoint({ x: 0, y: 0 }),
-          curveTo: stringifyPoint({ x: 0, y: 0 }),
-          hasCurveFrom: false,
-          hasCurveTo: false,
+          curveFrom: point,
+          curveTo: point,
+          hasCurveFrom: true,
```

Comment:
I have these curveFrom and curveTo set to true by default to render the control point curves. Would this be better as a point action in the point reducer?

Reply:
Yeah, I think so, otherwise we won't be able to do the right thing when switching the point's curve mode for the first time. I think it can fit into our existing actions, so we won't need to make a new one.